number_test.go

```diff
@@ -315,25 +303,13 @@ func TestNumber_InDelta(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			reporter := newMockReporter(t)
 
-			if tc.isInDelta {
-				NewNumber(reporter, tc.number).
-					InDelta(tc.value, tc.delta).
-					chain.assertNotFailed(t)
-			} else {
-				NewNumber(reporter, tc.number).
-					InDelta(tc.value, tc.delta).
-					chain.assertFailed(t)
-			}
+			NewNumber(reporter, tc.number).
+				InDelta(tc.value, tc.delta).
+				chain.assertOK(t, tc.isInDelta)
```

Comment:
nit: could you please reformat everything to a more compact 2-line format like this:

```go
			NewNumber(reporter, tc.number).InDelta(tc.value, tc.delta).
				chain.assertOK(t, tc.isInDelta)
```

Reply:
thsnk you for athe comment.
reformated!

Reply:
Sorry, I actually meant to switch from 3-line to 2-line format, not vise versa :)

I've pushed a commit: https://github.com/gavv/httpexpect/pull/347/commits/88fd692a0d3dc9dedb12d00157e65da147ff1ab6